docs/checks/internal/checks.yaml

```diff
@@ -691,14 +691,20 @@ checks:
     description: |
       Risk: `High`  (known vulnerabilities)
 
-      This check determines whether the project has open, unfixed vulnerabilities
-      using the [OSV (Open Source Vulnerabilities)](https://osv.dev/) service. An open
-      vulnerability is readily exploited by attackers and should be fixed as soon as
+      This check determines whether the project has open, unfixed vulnerabilities 
+      in it's dependencies using the [OSV (Open Source Vulnerabilities)](https://osv.dev/) service.
+      An open vulnerability is readily exploited by attackers and should be fixed as soon as
       possible.
     remediation:
       - >-
         Fix the vulnerabilities. The details of each vulnerability can be found
         on <https://osv.dev>.
+      - >-
+        If you believe the vulnerability does not actually affect your project, the 
+        vulnerability can be ignored by creating a `osv-scanner.toml` file next to the 
```

Comment:
Consider breaking into two sentences for simplicity/easy reading: "the vulnerability can be ignored. To ignore, create an [...] and specify the..."